README.md

```diff
@@ -7,47 +7,50 @@
     <li><a href="#scala_library">scala_library/scala_macro_library</a></li>
     <li><a href="#scala_binary">scala_binary</a></li>
     <li><a href="#scala_test">scala_test</a></li>
+    <li><a href="#scala_repl">scala_repl</a></li>
+    <li><a href="#scala_library_suite">scala_library_suite</a></li>
+    <li><a href="#scala_test_suite">scala_test_suite</a></li>
+    <li><a href="#thrift_library">thrift_library</a></li>
     <li><a href="#scalapb_proto_library">scalapb_proto_library</a></li>
+    <li><a href="#scala_toolchain">scala_toolchain</a></li>
   </ul>
 </div>
 
 ## Overview
 
-This rule is used for building [Scala][scala] projects with Bazel. There are
-currently four rules, `scala_library`, `scala_macro_library`, `scala_binary`
-and `scala_test`.
+These rules are used for building [Scala][scala] projects with Bazel. There are
+currently rules for building binaries (and repl apps), libraries, tests, and
+protobuf and thrift libraries.
 
 ## Getting started
 
-In order to use `scala_library`, `scala_macro_library`, and `scala_binary`,
-you must have bazel 0.5.3 or later and add the following to your WORKSPACE file:
+In order to use these rules you must have bazel 0.5.3 or later and add the
```

Comment:
We actually require bazel 0.23 by now. Any chance you can fix this as well? Or just drop this part since we have the table below to say from which commit we require which bazel version

Reply:
sure, let me get this too!